src/components/vm/overview/vmOverviewCard.jsx

```diff
@@ -180,6 +180,11 @@ class VmOverviewCard extends React.Component {
                             {_("General")}
                         </Text>
 
+                        <DescriptionListGroup>
+                            <DescriptionListTerm>{_("Connection")}</DescriptionListTerm>
+                            <DescriptionListDescription id={`${idPrefix}-connection`}>{vm.connectionName}</DescriptionListDescription>
```

Comment:
Thanks fo this PR. This needs to be translated.
Also To show some libvirt values in the UI, we use [`rephraseUI` function in the from src/helpers.js](https://github.com/cockpit-project/cockpit-machines/blob/main/src/helpers.js#L281). What it does is that it will turn it into more human friendly strings, and also **translate them**. E.g. "session" becomes "User session" (plus translation into the user's language)
```suggestion
                            <DescriptionListDescription id={`${idPrefix}-connection`}>{rephraseUI('connections', vm.connectionName)}</DescriptionListDescription>
```

Reply:
Thanks, will re-push it.